homeassistant/components/foscam/camera.py

```diff
@@ -208,7 +195,7 @@ def enable_motion_detection(self):
 
             self._motion_status = True
         except TypeError:
-            _LOGGER.debug("Communication problem")
+            LOGGER.debug("Communication problem")
```

Comment:
Updated this. I've been checking (as this was already there when I did my changes, even the first time), and looks like this error is because the library looks like doesn't check if the camera has the option (and looks like there is no way either) so this error can trigger if it tries to change the value over a `None` result.